pipeline/extract.py

```diff
@@ -88,21 +88,15 @@ def get_website_from_url(url: str) -> str:
 
 
 def scrape_pricing_process(html_content: bytes, url: str, product_id: int) -> dict:
-    """ Chooses which scraper to use based off of the URL """
+    """Chooses which scraper to use based off of the URL."""
 
     website_url = get_website_from_url(url)
 
     if "https://store.steampowered.com" in website_url:
         return scrape_from_steam_html(html_content, url, product_id)
 
-    # if "https://www.amazon.com" in url or "https://www.amazon.co." in website_url:
-    #     return scrape_from_amazon_html(html_content, url, product_id)
-
-    if "https://www.debenhams.com" in website_url:
-        return scrape_from_debenhams_html(html_content, url, product_id)
-
     logging.error(
-        "Cannot scrape that URL, since it's not an Amazon/Steam webpage.")
+        "Cannot scrape that URL, since it's not a Debenhams/Steam/John Lewis webpage.")
```

Comment:
Good idea to change the error message to reflect what pages we're scraping, but do we need the part referencing John Lewis?